snippets/csharp/VS_Snippets_CLR_System/system.guid.parse/cs/tryparseex1.cs

```diff
@@ -20,10 +20,12 @@ public static void Main()
             Console.WriteLine("Unable to convert {0} to a Guid", 
                               stringGuid);
       }                                      
+
```

Comment:
line 15
remove

line 17
if (Guid.TryParse(stringGuid, out var newGuid))

line 18
Console.WriteLine($"Converted {stringGuid} to a Guid");

line 20
Console.WriteLine($"Unable to convert {stringGuid} to a Guid");

Reply:
Done.